src/main/java/web/Calculator.java

```diff
@@ -0,0 +1,102 @@
+package web;
+
+import java.util.*;
+import java.util.regex.Pattern;
+
+class Calculator{
+    private static int calculate (int num1, int num2, char op) {
+        int number1 = num2;
+        int number2 = num1;
+        switch (op) {
+            case '*':
+                return number1 * number2;
+            case '/':
+                return number1 / number2;
+            case '+':
+                return number1 + number2;
+            case '-':
+                return number1 - number2;
+            default:
+                return 0;
+        }
+    }
+
+    private Map<String, String> variables;
+
+    Calculator(Map<String, String> variables) {
+        this.variables = variables;
+    }
+
+    int solve(String equation) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4846841)